README.md

```diff
@@ -820,32 +838,55 @@ Model Definition is an object describing an entity. It has the following structu
     /*
      * Custom operations available in CRUD Editor.
      * An operation handler is called by pressing a dedicated button.
+	 * No arguments are passed to the method in Create View
+	 * since it does not have persistent instance.
+	 *
+	 * In case of unsaved changes, Confirmation Dialog is called
+	 * after dedicated button press and before handler() call
+	 * => each external operation must return new view name/state,
+	 *    or set disabled to true,
+	 *    or set show to false
+	 * - othersise calling Confirmation Dialog is in vain.
      */
-    ?operations: function(<object, entity persistent instance>, {
-      name: <string, View name>,  // See EditorComponent props.view.name
-      state: <object, Full View State>  // See EditorComponent props.view.state
-    }) {
+    ?customOperations: function(<object, entity persistent instance> ) {
       ...
       return [{
-        name: <string, operation ID>,
-
-        /*
-         * name of an icon to be displayed inside a button, ex. "trash", "edit";
-         * see full list at
-         * http://getbootstrap.com/components/#glyphicons
-         */
-        ?icon: <string>,
-
-        /*
-         * The operation button is not displayed if there is no hanlder.
-         */
-        ?handler() {
+        handler() {
           ...
           // return value is either undefined or view name/state.
           return {
             name: <string, View Name>,
             ?state: <object, View State, empty object by default>
           };
+        },
+        ui({
+          name: <string, View name>,  // See EditorComponent props.view.name
+          state: <object, Full View State>  // See EditorComponent props.view.state
+        }) {
+          return {
+            title() {
+              ...
+              return <string, custom operation translated title>,
+            },
+
+            ?show: <boolean, true by default>,
+            ?disabled: <boolean, false by default>,
+
+            /*
+             * whether the operation has own dedicated button (false)
+             * or it is to be placed in a dropdown of a previous button (true).
+             * A previous button is either previous custom operation with "dropdown" set to false
+             * OR (for Search View) "Edit" button if there is no such custom operation.
+             */
+            ?dropdown: <boolean, true by default>,
+
+            /*
+             * name of an icon to be displayed inside a button, ex. "trash", "edit";
+             * see full list at
+             * http://getbootstrap.com/components/#glyphicons
+             */
+            ?icon: <string>
```

Comment:
TBD: should there be an ability to define a component? It gives a possibility to define e.g. font-awesome icons and doesn't force usage of bootstrap' glyphicons.